target/linux/mediatek/patches-6.12/117-complete-mt7981b-dtsi.patch

```diff
@@ -433,21 +434,6 @@ working:
 +			};
 +		};
 +
-+		wdma: wdma@15104800 {
-+			compatible = "mediatek,wed-wdma";
-+			reg = <0 0x15104800 0 0x400>,
-+			      <0 0x15104c00 0 0x400>;
```

Comment:
These address ranges are already hardcoded in the upstream driver.
https://github.com/torvalds/linux/blob/dd9b004b7ff3289fb7bae35130c0a5c0537266af/drivers/net/ethernet/mediatek/mtk_eth_soc.c#L155-L158